### PR TITLE
Fixes aliens

### PR DESCRIPTION
**Pull request by Bod9001**
### Changelog:

CL: [Fix] Fixed Aliens that spawn without a brain organ dying immediately
CL: [Fix] Fixed Aliens walking incredibly slow

### CODE REVIEW

**Comment by MaxIsJoe** [comment]
changelogs need to be a bit more verbose. What did you fix?

**Comment by corp-0** [comment]
aliens

**Comment by MaxIsJoe** [comment]
what happened to that github PR ai experiment thingy? maybe we can enable it so we avoid things like this

**Comment by MaxIsJoe** [comment]
CL: [Fix] Fixed Aliens that spawn without a brain organ dying immediately 
CL: [Fix] Fixed Aliens walking incredibly slow due to them now having a leg body part.

**Comment by corp-0** [comment]
> what happened to that github PR ai experiment thingy? maybe we can enable it so we avoid things like this

they haven't released it yet. We are in the waiting list.

File: UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs [hunk 1]
```diff
@@ -1001,16 +1004,20 @@ public void CalculateOverallHealth()
...
+				{
+					currentHealth -= 200;
+					healthStateController.SetOverallHealth(currentHealth);
+					CheckHeartStatus();
+					return;
+				}
+				else
+				{
+
+					currentHealth -= brain.RelatedPart.Oxy;
```
Comment (MaxIsJoe) [inline UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs:1019]: why are we subtracting oxygen?

Comment (Bod9001) [reply UnityProject/Assets/Scripts/HealthV2/Living/LivingHealthMasterBase.cs:1019]: It's indicates that your brain is taking oxygen damage

**Comment by Bod9001** [comment]
aliens don't have limbs or body parts, they are hardcoded As Hell

**Pull request merged**